### PR TITLE
Organize state

### PR DESCRIPTION
**Pull request by dcposch**
@feross @ngoldman

### CODE REVIEW

**Comment by dcposch** [comment]
Merging because this is gonna get conflicts quickly otherwise. Would still appreciate review!

File: renderer/views/torrent-list.js [hunk 1]
```diff
@@ -6,18 +6,18 @@ var hx = hyperx(h)
 var prettyBytes = require('pretty-bytes')
 
 function TorrentList (state, dispatch) {
-  var torrents = state.view.client
-    ? state.view.client.torrents
+  var torrents = state.client
+    ? state.client.torrents
```
Comment (feross) [inline renderer/views/torrent-list.js]: this seems to have come back from a merge, thought i deleted this. It's not necessary. `state.client.torrents` will always exist

File: renderer/views/player.js [hunk 2]
```diff
@@ -112,9 +111,7 @@ function renderPlayerControls (state, dispatch) {
 
   // Handles a click or drag to scrub (jump to another position in the video)
   function handleScrub (e) {
-    // TODO: don't use remote -- it does IPC with the main process which is overkill
-    // just to get the width
-    var windowWidth = electron.remote.getCurrentWindow().getBounds().width
+    var windowWidth = document.querySelector('body').clientWidth
```
Comment (feross) [inline renderer/views/player.js]: nice!

**Comment by feross** [comment]
LGTM

File: renderer/views/torrent-list.js (hunk 1 above)
Comment (feross) [inline renderer/views/torrent-list.js]: will fix

**Pull request merged**